internal/pipe/release/release.go

```diff
@@ -106,7 +106,9 @@ func (Pipe) Publish(ctx *context.Context) error {
 	if err := doPublish(ctx, c); err != nil {
 		return err
 	}
-	log.WithField("url", ctx.ReleaseURL).Info("published")
+	if !ctx.Config.Release.Draft {
+		log.WithField("url", ctx.ReleaseURL).Info("published")
+	}
```

Comment:
maybe instead do something like this:
```suggestion
	log.WithField("url", ctx.ReleaseURL).WithField("published", !ctx.Config.Release.Draft).Info("release created/updated")
```